src/app.rs

```diff
@@ -254,6 +262,18 @@ fn usage() -> HashMap<&'static str, Help> {
     doc!(h, "rg-alias-hidden-ignore"
         , "Alias for no-ignore and/or hidden"
         , "Alias for no-ignore ('u') and no-ignore and hidden ('uu')");
-
+    doc!(h, "size"
+        , "Limit results based on the size of files."
+        , "Limit results based on the size of files using the format <+-><NUM><UN>.\n   \
+            '+': file size must be greater than this\n   \
+            '-': file size must be less than this\n   \
+            'NUM':      The numeric size (e.g. 500)\n   \
+            'UN':       The units for NUM.\n\
```

Comment:
Great help text!

Can we replace `<UN>` by `<UNIT>`? I think we can be verbose in the "long" help text.